### PR TITLE
Feature/custom titles for auth forms

### PR DESCRIPTION
**Pull request by illyawebdev**

### CODE REVIEW

File: src/managers/form/buttons/TriggerButton.js [hunk 1]
```diff
@@ -1,30 +1,160 @@
...
+    subscribeEventForChanges() {
+        /** change Firebase button titles on load */
+        this.events.subscribe(FIREBASE_UI_SHOWN, () => {
+            this.#changeAuthButtonsTitle(this.#options.defaultFormSlug, false);
+            this.#changeAuthButtonsTitleColor();
+        }, {once: true});
+
+        /** change titles on auth form change */
+        this.events.subscribe(DEFAULT_AUTH_FORM_SLUG_UPDATED, (value) => {
+            if (value?.new && value.new !== value.old) {
```
Comment (tsybulskyserg) [inline src/managers/form/buttons/TriggerButton.js:102]: do we need checking value?.new ?

Comment (tsybulskyserg) [inline src/managers/form/buttons/TriggerButton.js:102]:
```javascript
const textColor = get(this.#options[aProvider], 'textColor');
if(!textColor) continue;
```

Comment (illyawebdev) [reply src/managers/form/buttons/TriggerButton.js]: Done: Added lodash.get and rewrote part of the code using it

Comment (illyawebdev) [reply src/managers/form/buttons/TriggerButton.js]: Done.

**Pull request merged**